libs/go/libp2p_node/dht/dhtclient/dhtclient.go

```diff
@@ -466,10 +485,10 @@ func (dhtClient *DHTClient) RouteEnvelope(envel *aea.Envelope) error {
 		Str("target", target).
 		Msg("requesting agent record from relay...")
 
-	lookupRequest := &dhtnode.LookupRequest{AgentAddress: target}
-	msg := &dhtnode.AcnMessage{
-		Version: dhtnode.CurrentVersion,
-		Payload: &dhtnode.AcnMessage_LookupRequest{LookupRequest: lookupRequest},
+	lookupRequestPerformative := &LookupRequestPerformative{AgentAddress: target}
+	msg := &AcnMessage{
+		// Version: dhtnode.CurrentVersion,
```

Comment:
Not sure if it makes sense to include the version here. What was the thinking behind this @lrahmani ? The protocol itself is versioned.

Reply:
@marcofavorito & @solarw

Reply:
version field left from previous protocol implementation, so not needed, cause protocol spec version is now included

Reply:
Yes lets remove the version field everywhere please